Fitbit.Portable/Models/HeartActivitiesIntradayTimeSeries.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Fitbit.Models;
+using Newtonsoft.Json;
+
+namespace Fitbit.Api.Portable.Models
+{
+    public class HeartActivitiesIntradayTimeSeries
+    {
+        [JsonProperty(PropertyName = "activities-heart")]
+        public List<ActivitesHeart> ActivitiesHeart { get; set; }
+        [JsonProperty(PropertyName = "activities-heart-intraday")]
+        public ActivitesHeartIntraday ActivitiesHeartIntraday { get; set; }
+    }
+    public class ActivitesHeart
```

Comment:
All classes should be in their own files

Reply:
👍